### PR TITLE
bump tww talents + re-enable resource inclusion

### PR DESCRIPTION
**Pull request by emallson**
RAIDBOTS data has been improved so this no longer needs the hacks to determine class from spec.

as a result of this import, i had to disable several additional specs:

- Fury Warrior
- BM Hunter
- Survival Hunter
- MM Hunter
- Holy Priest
- Disc Priest

none of the specs that has been re-enabled for TWW was broken by this change (though I think @trevorm4 added some mana values that might be overwritten?)

### CODE REVIEW

File: src/common/TALENTS/priest.ts [hunk 1]
```diff
@@ -2,14 +2,6 @@
...
 
 const talents = {
-  ABOMINATION_LIMB_TALENT: {
-    id: 383269,
-    name: 'Abomination Limb',
-    icon: 'ability_maldraxxus_deathknight',
-    maxRanks: 1,
-    entryIds: [96177],
-    definitionIds: [{ id: 101179, specId: 258 }],
-  },
```
Comment (emallson) [inline src/common/TALENTS/priest.ts:12]: if anyone is curious why the priest file is -4,102 lines, its stuff like this from the previous bad raidbots data

Comment (ToppleTheNun) [reply src/common/TALENTS/priest.ts:12]: HUH

Comment (emallson) [reply src/common/TALENTS/priest.ts:12]: there was a bunch of duplicated data showing the shadow priest spec id. @ljosberinn knows all about it

Comment (ljosberinn) [reply src/common/TALENTS/priest.ts:12]: shadow is so meta, even rogue and dk got a new shadow spec (and others)

**Comment by trevorm4** [comment]
I didnt add any mana values because i dont have access to beta so couldnt check mana costs :P so nothing to worry about there

**Pull request merged**